apps/rush-lib/src/api/Rush.ts

```diff
@@ -63,12 +103,67 @@ export class Rush {
     return PackageJsonLookup.loadOwnPackageJson(__dirname).version;
   }
 
+  private static _tryGetRushConfiguration(): RushConfiguration | undefined {
+    try {
+      if (RushConfiguration.tryFindRushJsonLocation()) {
+        return RushConfiguration.loadFromDefaultLocation({ showVerbose: true });
+      }
+    } catch (e) {
+      // ignore
+    }
+
+    return undefined;
+  }
+
+  /**
+   * This function normalizes legacy options to the current {@see ILaunchOptions} object.
+   */
+  private static _normalizeLaunchOptions(arg: ILaunchOptions): ILaunchOptions {
+    return (typeof arg === 'boolean')
+      ? { isManaged: arg } // In older versions of Rush, this the `launch` functions took a boolean arg for "isManaged"
+      : arg;
+  }
+
   private static _printStartupBanner(isManaged: boolean): void {
+    const nodeVersion: string = process.versions.node;
+    const nodeMajorVersion: number = semver.major(nodeVersion);
+    const nodeReleaseLabel: string = (nodeMajorVersion % 2 === 0)
+      ? (!!(process as IExtendedNodeProcess).release.lts ? 'LTS' : 'pre-LTS')
+      : 'unstable';
+
     console.log(
       EOL +
       colors.bold(`Rush Multi-Project Build Tool ${Rush.version}` + colors.yellow(isManaged ? '' : ' (unmanaged)')) +
       colors.cyan(` - ${RushConstants.rushWebSiteUrl}`) +
+      EOL +
+      `Node.js version is ${nodeVersion} (${nodeReleaseLabel})` +
       EOL
     );
   }
+
+  private static _warnAboutNodeVersion(rushConfiguration: RushConfiguration | undefined): void {
+    const nodeVersion: string = process.versions.node;
+
+    if (semver.satisfies(nodeVersion, '>= 11.0.0')) {
+      console.log();
+      console.warn(colors.yellow(
+        `Your version of Node.js (${nodeVersion}) has not been tested with this release` +
+        `of the Rush engine.  Please consider upgrading the "rushVersion" setting in rush.json, ` +
+        `or  downgrading Node.js.`
+      ));
+      console.log();
+    } else if (
+      rushConfiguration &&
+      !rushConfiguration.suppressNodeLtsWarning &&
+      !(process as IExtendedNodeProcess).release.lts
+    ) {
+      console.log();
+      console.warn(colors.yellow(
+        `Your version of Node.js (${nodeVersion}) has an odd major version number. ` +
```

Comment:
@bterlson pointed out that the meaning of "odd" is unclear here.

Let's change the wording to:  `Your version of Node.js (${nodeVersion}) is an odd-numbered release. `